### DISCUSSION TITLE
MikuMikuDance model rendered using OpenGL ES.

### DISCUSSION THREAD

**Discussion by qian-o** [dotnet/Silk.NET#1661] [Show and tell]
Hello everyone, I have created an example using [Silk.NET](https://github.com/dotnet/Silk.NET) to render MMD animations and simulate physics in real time. This example was inspired by the famous [saba](https://github.com/benikabocha/saba) repository and implemented as a simulation. Due to my love for C#, I decided to bring it over. If you are interested, you can check out the original [saba](https://github.com/benikabocha/saba) repository.

SabaSharp : https://github.com/qian-o/SabaSharp
![image](https://github.com/dotnet/Silk.NET/assets/84434846/e0b0d371-4778-4ad2-93a2-c57c413ad05d)
![image](https://github.com/dotnet/Silk.NET/assets/84434846/de438dc9-905c-4ae9-9107-4b033a890d0b)

**Comment by Perksey** [comment]
Wow, this is amazing!!
